controllers/activemqartemis_reconciler.go

```diff
@@ -818,6 +818,9 @@ func (reconciler *ActiveMQArtemisReconcilerImpl) configureConsoleExposure(custom
 		Name:      customResource.Name,
 		Namespace: customResource.Namespace,
 	}
+	commonPortName := "wconsj"
+	secondPort := int32(8261)
+	portNumber := int32(8161)
```

Comment:
if the port name with the ordinal will be deprecated should the port 8161 have the name `wconsj`?
```suggestion
	commonPortName := "wconsj"
	secondPort := int32(8161)
	portNumber := int32(8162)
```